### PR TITLE
Fixes #13676 - Creating sub package for ostree

### PR DESCRIPTION
**Pull request by parthaa**

### CODE REVIEW

File: rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -95,6 +95,14 @@ Summary:    Documentation for rubygem-%{gem_name}
 %description doc
 This package contains documentation for rubygem-%{gem_name}.
 
+%package ostree
+Requires:   %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Summary:    Katello Ostree Plugin
+%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}-ostree}
```
Comment (ehelms) [inline rubygem-katello/rubygem-katello.spec]: This line isn't needed, there is no ruby193-rubygem-katello-ostree

File: rubygem-katello/rubygem-katello.spec [hunk 2]
```diff
@@ -95,6 +95,14 @@ Summary:    Documentation for rubygem-%{gem_name}
 %description doc
 This package contains documentation for rubygem-%{gem_name}.
 
+%package ostree
```
Comment (ehelms) [inline rubygem-katello/rubygem-katello.spec]: I think we need to custom name this as RPM will try to use `-` to connect, and we want to use the more typical trend of `_`:

```
tfm-rubygem-katello-ostree
```

vs

```
tfm-rubygem-katello_ostree
```

File: rubygem-katello/rubygem-katello.spec [hunk 3]
```diff
@@ -95,6 +95,14 @@ Summary:    Documentation for rubygem-%{gem_name}
...
+Requires:   %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Summary:    Katello Ostree Plugin
+%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}-ostree}
+
+%description ostree
+This package contains documentation for ostree plugin for rubygem-%{gem_name}.
```
Comment (ehelms) [inline rubygem-katello/rubygem-katello.spec]: Documentation for ostree plugin? I think it does more than that :)

File: rubygem-katello/rubygem-katello.spec (hunk 2 above)
Comment (parthaa) [inline rubygem-katello/rubygem-katello.spec]: do we do this anywhere else ?

Comment (parthaa) [inline rubygem-katello/rubygem-katello.spec]: I can try to do something like `%package -n  %{?scl_prefix}rubygem-%{gem_name}_ostree`

Comment (ehelms) [inline rubygem-katello/rubygem-katello.spec]: :+1: -- @jlsherrill do you agree?

Comment (jlsherrill) [inline rubygem-katello/rubygem-katello.spec]: :+1: i think that makes sense

Comment (parthaa) [inline rubygem-katello/rubygem-katello.spec]: @jlsherrill @ehelms updated

**Comment by ehelms** [comment]
You'll want to add this to the comps as well.

**Comment by parthaa** [comment]
> You'll want to add this to the comps as well.

Good call. Updated.

**Comment by parthaa** [comment]
We need this guy in too https://github.com/Katello/katello-packaging/pull/165 ..

**Comment by jlsherrill** [comment]
ACK from me, build looks good

**Comment by ehelms** [comment]
:+1:

**Pull request merged**